packages/sign/src/__tests__/external/error.test.ts

```diff
@@ -40,13 +40,13 @@ describe('checkStatus', () => {
       ok: false,
     });
 
-    it('throws an error', () => {
+    it('throws an error', async () => {
       expect.assertions(2);
       try {
-        checkStatus(response);
+        await checkStatus(response);
       } catch (e) {
         assert(e instanceof HTTPError);
-        expect(e.message).toEqual('HTTP Error: 404 Not Found');
+        expect(e.message).toEqual('(404) Not Found');
```

Comment:
Do we want any tests that check the entire error? Looks like these tests are only checking the checkStatus/http error and not the full internal error like: `error creating signing certificate - (400) There was an error processing the identity token`

Reply:
Good call. Added more tests.